src/main/java/org/zalando/nakadi/service/AdminService.java

```diff
@@ -62,6 +64,13 @@ public boolean isAdmin(final AuthorizationService.Operation operation) {
         return authorizationService.isAuthorized(operation, resource);
     }
 
+    public boolean hasAllDataAccess(final AuthorizationService.Operation operation) {
+        final List<Permission> permissions = authorizationDbRepository.listAllDataAccess();
```

Comment:
I don't remember: have we planned to have simple guava cache in this implementation or not?
I checked - currently reading events endpoint (both hila and lola) is called ~25 times/sec.

Reply:
Good point. Here is a very simple cache, where data expires after 1 minute, as we discussed: 625c2e2